infoblox_client/utils.py

```diff
@@ -345,3 +345,16 @@ def safe_json_load(data):
         return jsonutils.loads(data)
     except ValueError:
         LOG.warn("Could not decode reply into json: %s", data)
+
+
+def try_value_to_bool(value):
+    """Tries to convert value into boolean.
+
+    If value is not a string representation of True or False,
+    then unchanged incoming value is returned.
+    """
+    if value == 'True':
+        return True
+    elif value == 'False':
```

Comment:
I see, then I'll add lest strict checking mode allowing the same values as [1] does.

[1] https://github.com/infobloxopen/infoblox-netmri/blob/master/infoblox_netmri/__init__.py#L33